### PR TITLE
Application Gateway command-lets

### PR DESCRIPTION
**Pull request by kagamsft**

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@kagamsft__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (kaga). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

**Comment by ogail** [comment]
@azuresdkci add to whitelist

File: src/Common/Commands.ScenarioTest/Commands.ScenarioTest.csproj [hunk 1]
```diff
@@ -174,6 +174,7 @@
     <Compile Include="Common\TestEnvironment.cs" />
     <Compile Include="Common\TestEnvironmentFactory.cs" />
     <Compile Include="Common\TokenCloudCredentialsHelper.cs" />
+    <None Include="app.config" />
```
Comment (ogail) [inline src/Common/Commands.ScenarioTest/Commands.ScenarioTest.csproj]: Remove all `app.config` from all projects

File: src/ServiceManagement/Network/Commands.Network/ApplicationGateway/AddApplicationGatewaySslCertificate.cs [hunk 2]
```diff
@@ -0,0 +1,40 @@
+﻿using System;
```
Comment (ogail) [inline src/ServiceManagement/Network/Commands.Network/ApplicationGateway/AddApplicationGatewaySslCertificate.cs]: Add copyright headers for all files

File: src/ServiceManagement/StorSimple/Commands.StorSimple.Test/App.config [hunk 3]
```diff
@@ -8,4 +8,12 @@
     <add key="CisStampId" value="7ef900f6-ad91-497d-a9c5-2742b7ed4a75" />
     <add key="ClientSettingsProvider.ServiceUri" value="" />
   </appSettings>
+  <runtime>
```
Comment (ogail) [inline src/ServiceManagement/StorSimple/Commands.StorSimple.Test/App.config]: Please revert all changes unrelated to your project

File: src/ServiceManagement/Network/Commands.Network/Microsoft.WindowsAzure.Commands.ApplicationGateway.dll-Help - Copy.xml [hunk 4]
```diff
@@ -0,0 +1,1542 @@
+﻿<?xml version="1.0" encoding="utf-8"?><helpItems xmlns="http://msh" schema="maml">
```
Comment (ogail) [inline src/ServiceManagement/Network/Commands.Network/Microsoft.WindowsAzure.Commands.ApplicationGateway.dll-Help - Copy.xml]: Remove copy file

**Comment by ogail** [comment]
Why the help is splitted between two files?

**Pull request merged**